packages/runtime-node/src/node-env-manager.ts

```diff
@@ -78,11 +84,16 @@ export class NodeEnvManager {
 
         const host = new WsServerHost(socketServer);
 
-        this.autoLaunchDispose = async () => {
-            host.dispose();
-            disposeListener();
-            await close();
-        };
+        this.disposables.registerGroup('auto launch', { before: 'default' });
```

Comment:
why group here? it will dispose after every time because we register in constructor.